test/e2e/data/infrastructure-aws/kustomize_sources/peered-remote/patches/limit-az-order.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AWSCluster
+metadata:
+  name: "${CLUSTER_NAME}"
+spec:
+  network:
+    vpc:
+      availabilityZoneSelection: Ordered
```

Comment:
This field is only used for selecting which AZs we auto-create subnets on for managed VPCs.